src/main/java/teammates/common/datatransfer/attributes/StudentProfileAttributes.java

```diff
@@ -161,4 +159,63 @@ public void sanitizeForSaving() {
         this.googleId = SanitizationHelper.sanitizeGoogleId(this.googleId);
     }
 
+    /**
+     * A Builder class for {@link StudentProfileAttributes}.
+     */
+    public static class Builder {
+        private final StudentProfileAttributes profileAttributes = new StudentProfileAttributes();
+
+        public Builder withGoogleId(String googleId) {
+            profileAttributes.googleId = googleId;
```

Comment:
I am aware that this was not handled before, but any chance that the passed in parameter can be `null` and lead to unexpected behavior?
(applies to all the other fields as well)